docs/src/checklists.md

```diff
@@ -0,0 +1,38 @@
+# Checklists
+
+The purpose of this page is to collate a series of checklists for commonly
+performed changes to the source code of Documenter.
+
+In each case, copy the checklist into the description of the pull request.
+
+## Making a release
+
+In preparation for a release, use the following checklist.
+
+````
+## Pre-release
+
+ - [ ] Change the version number in `Project.toml`
+        * If the release is breaking, increment MAJOR
+        * If the release adds a new user-visible feature, increment MINOR
+        * Otherwise (bug-fixes, documentation improvements), increment PATCH
+ - [ ] Update `CHANGELOG.md`, following the existing style (in particular, make
+       sure that the change log for this version has the correct version number
+       and date).
+ - [ ] Run `make changelog`, to make sure that all the issue references in
+       `CHANGELOG.md` are up to date.
+ - [ ] Check that the commit messages in this PR do not contain `[ci skip]`
+
+## The release
+
+ - [ ] After merging this pull request, tag the release. There are two options for this:
+
+      1. [Comment `[at]JuliaRegistrator register` on the GitHub commit.](https://github.com/JuliaRegistries/Registrator.jl#via-the-github-app)
```

Comment:
Perhaps add TagBot release notes here? A link to the changelog at least.

Reply:
I added that into `TagBot.yml`: https://github.com/JuliaDocs/Documenter.jl/blob/efb3ab4d176023b50fe5d6c49a3d87096d67760c/.github/workflows/TagBot.yml#L19-L28 and it seems to generate correct release notes: https://github.com/JuliaDocs/Documenter.jl/releases

Reply:
Ah cool, didn't know that was possible!